src/_common/game/thumbnail-img/thumbnail-img.vue

```diff
@@ -6,25 +6,27 @@
 		}"
 	>
 		<div class="-inner">
-			<app-jolticon class="-icon" icon="game" />
+			<app-img-backdrop :item="mediaItem" radius="8px">
+				<app-jolticon class="-icon" icon="game" />
```

Comment:
Make the icon white now with some sort of opacity so that it blends into the colors of the backdrop.